discovery-provider/src/queries/get_user_with_wallet.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from src.models.models import User
+from src.utils import db_session
+
+logger = logging.getLogger(__name__)
+
+
+def get_user_with_wallet(wallet):
```

Comment:
could add types if you wanted ie
`def get_user_with_wallet(wallet: str) -> int`

Reply:
:⭕